### PR TITLE
0.6.1 release

### PR DESCRIPTION
**Pull request by gerashegalov**
Bug fixes:
- Ensure correct metrics despite model failures on some CV folds [#404](https://github.com/salesforce/TransmogrifAI/pull/404)
- Fix flaky `ModelInsight` tests [#395](https://github.com/salesforce/TransmogrifAI/pull/395)
- Avoid creating `SparseVector`s for LOCO [#377](https://github.com/salesforce/TransmogrifAI/pull/377)

New features / updates:
- Model combiner [#385](https://github.com/salesforce/TransmogrifAI/pull/399)
- Added new sample for HousingPrices [#365](https://github.com/salesforce/TransmogrifAI/pull/365)
- Test to verify that custom metrics appear in model insight metrics [#387](https://github.com/salesforce/TransmogrifAI/pull/387)
- Add `FeatureDistribution` to `SerializationFormat`s [#383](https://github.com/salesforce/TransmogrifAI/pull/383)
- Add metadata to `OpStandadrdScaler` to allow for descaling [#378](https://github.com/salesforce/TransmogrifAI/pull/378)
- Improve json serde error in `evalMetFromJson` [#380](https://github.com/salesforce/TransmogrifAI/pull/380)
- Track mean & standard deviation as metrics for numeric features and for text length of text features [#354](https://github.com/salesforce/TransmogrifAI/pull/354)
- Making model selectors robust to failing models [#372](https://github.com/salesforce/TransmogrifAI/pull/372)
- Use compact and compressed model json by default [#375](https://github.com/salesforce/TransmogrifAI/pull/375)
- Descale feature contribution for Linear Regression & Logistic Regression [#345](https://github.com/salesforce/TransmogrifAI/pull/345)

Dependency updates:   
- Update tika version [#382](https://github.com/salesforce/TransmogrifAI/pull/382)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+- Add metadata to `OpStandadrdScaler` to allow for descaling [#378](https://github.com/salesforce/TransmogrifAI/pull/378)
+- Improve json serde error in `evalMetFromJson` [#380](https://github.com/salesforce/TransmogrifAI/pull/380)
+- Track mean & standard deviation of text length as a metric for text feature [#354](https://github.com/salesforce/TransmogrifAI/pull/354)
+- Making model selectors robust to failing models [#372](https://github.com/salesforce/TransmogrifAI/pull/372)
+- Use compact and compressed model json by default [#375](https://github.com/salesforce/TransmogrifAI/pull/375)
+- Descale feature contribution for Linear Regression & Logistic Regression [#345](https://github.com/salesforce/TransmogrifAI/pull/345)
+
+Dependency updates:   
+- Update tika version [#382](https://github.com/salesforce/TransmogrifAI/pull/382)
+- Revert back to Spark 2.3 [#399](https://github.com/salesforce/TransmogrifAI/pull/399)
```
Comment (tovbinm) [inline CHANGELOG.md]: I think you can omit this one

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+- Avoid creating `SparseVector`s for LOCO [#377](https://github.com/salesforce/TransmogrifAI/pull/377)
+
+New features / updates:
+- Model combiner [#385](https://github.com/salesforce/TransmogrifAI/pull/399)
+- Added new sample for HousingPrices [#365](https://github.com/salesforce/TransmogrifAI/pull/365)
+- Test to verify that custom metrics appear in model insight metrics [#387](https://github.com/salesforce/TransmogrifAI/pull/387)
+- Add `FeatureDistribution` to `SerializationFormat`s [#383](https://github.com/salesforce/TransmogrifAI/pull/383)
+- Add metadata to `OpStandadrdScaler` to allow for descaling [#378](https://github.com/salesforce/TransmogrifAI/pull/378)
+- Improve json serde error in `evalMetFromJson` [#380](https://github.com/salesforce/TransmogrifAI/pull/380)
+- Track mean & standard deviation of text length as a metric for text feature [#354](https://github.com/salesforce/TransmogrifAI/pull/354)
```
Comment (leahmcguire) [inline CHANGELOG.md]: this is actually on all raw features not just text

Comment (gerashegalov) [reply CHANGELOG.md]: rephrased after checking back with @TuanNguyen27

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+- Making model selectors robust to failing models [#372](https://github.com/salesforce/TransmogrifAI/pull/372)
```
Comment (leahmcguire) [inline CHANGELOG.md]: also there is a PR to fix an issue with models that fail in some CV fold but not all that we should get in before the release here: https://github.com/salesforce/TransmogrifAI/pull/404

**Pull request merged**